test/TestApps/PushNotificationsDemoPackage/Package.appxmanifest

```diff
@@ -0,0 +1,65 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
+  xmlns:com="http://schemas.microsoft.com/appx/manifest/com/windows10"
+  IgnorableNamespaces="uap rescap">
+
+	<Identity
+	  Name="PushNotificationsWin32App"
+	  Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US"
+	  Version="1.0.0.0" />
+
+	<Properties>
+		<DisplayName>MSIXPackager</DisplayName>
+		<PublisherDisplayName>purifoypaul</PublisherDisplayName>
+		<Logo>Images\StoreLogo.png</Logo>
+	</Properties>
+
+	<Dependencies>
+		<TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.0.0" MaxVersionTested="10.0.0.0" />
+		<TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.14393.0" MaxVersionTested="10.0.14393.0" />
+		<PackageDependency Name="Microsoft.ProjectReunion.Framework" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" MinVersion="0.0.0.0"/>
+	</Dependencies>
+
+	<Resources>
+		<Resource Language="x-generate"/>
+	</Resources>
+
+	<Applications>
+		<Application Id="App"
+		  Executable="$targetnametoken$.exe"
+		  EntryPoint="$targetentrypoint$">
+			<uap:VisualElements
+			  DisplayName="MSIXPackager"
+			  Description="MSIXPackager"
+			  BackgroundColor="transparent"
+			  Square150x150Logo="Images\Square150x150Logo.png"
+			  Square44x44Logo="Images\Square44x44Logo.png">
+				<uap:DefaultTile Wide310x150Logo="Images\Wide310x150Logo.png" />
+				<uap:SplashScreen Image="Images\SplashScreen.png" />
+			</uap:VisualElements>
+			<Extensions>
+				<uap:Extension Category="windows.protocol" EntryPoint="Windows.FullTrustApplication">
+					<uap:Protocol Name="reuniontestprotocol-packaged">
+						<uap:DisplayName>ReunionTestProtocol</uap:DisplayName>
+					</uap:Protocol>
+				</uap:Extension>
+				<com:Extension Category="windows.comServer">
+					<com:ComServer>
+						<com:ExeServer Executable="PushNotificationsDemoApp\PushNotificationsDemoApp.exe" DisplayName="SampleBackgroundApp" Arguments="-ReunionPushServer">
+							<com:Class Id="ccd2ae3f-764f-4ae3-be45-9804761b28b2" DisplayName="Reunion Push" />
+						</com:ExeServer>
+					</com:ComServer>
+				</com:Extension>
+			</Extensions>
+		</Application>
+	</Applications>
+
+	<Capabilities>
+		<Capability Name="internetClient" />
+		<rescap:Capability Name="runFullTrust" />
```

Comment:
Nit: please reformat XML with 4-spaces instead of tabs.

@chrisguzak - can you help us figure out how to apply an editorconfig to enforce formatting rules?